docs/admin/cluster-management.md

```diff
@@ -33,43 +33,66 @@ Documentation for other releases can be found at
 
 # Cluster Management
 
+This document describes several topics related to the lifecycle of the cluster management software: creating a new cluster,
+upgrading your cluster's
+master and worker nodes, performing node maintainence (e.g. kernel upgrades) and upgrading the Kubernetes API version of a
+running cluster.
+
 ## Creating and configuring a Cluster
 
 To install Kubernetes on a set of machines, consult one of the existing [Getting Started guides](../../docs/getting-started-guides/README.md) depending on your environment.
 
 ## Upgrading a cluster
 
-The `cluster/kube-push.sh` script will do a rudimentary update; it is a 1.0 roadmap item to have a robust live cluster update system.
+The current state of cluster upgrades is provider dependent.  In particular both Google Container Engine (GKE) and
```

Comment:
Can you clarify in this section minion upgrades vs. master node upgrade? I would suggest having separate subsections for each, even if some of the information is redundant. Right now it's confusing, for example it's not obvious that the PD stuff only applies to master node, not minion.